src/content/docs/en/release/upgrade-to-ultramarine-40.mdx

```diff
@@ -17,7 +17,10 @@ You might need to update your system normally before the upgrade to 40 shows up
 
 ## Pantheon Edition
 
-TODO
+The Pantheon upgrade path isn't ready yet.
+While all of the required packages are in the UM and Terra repositories, the Pantheon desktop enviroment is broken for a variety of reasons.
+We are currently working with upstream and other partners to resolve this, but it's possible that Pantheon may be skipped, deprecated, or defered until mid-cycle.
+If you are interested in contributing to Ultramarine Pantheon, please join our [Discord](https://discord.gg/5fdPuxTg5Q) or [Matrix[](https://matrix.to/#/#hub:fyralabs.com) and let us know!
```

Comment:
say something like "our chats" and link the community page instead of the disc and matrix links

Reply:
can you add a suggestion?

Reply:
jade, add it as a suggestion in GitHub

Reply:
the only option is edit directly?

Reply:
```suggestion
If you are interested in contributing to Ultramarine Pantheon, please join our [Discord](https://discord.gg/5fdPuxTg5Q) or [Matrix[](https://matrix.to/#/#hub:fyralabs.com) and let us know!
```

Reply:
```suggestion
If you are interested in contributing to Ultramarine Pantheon, please join [our chats](en/community/community) and let us know!
```